src/Renderer.php

```diff
@@ -198,7 +198,7 @@ protected function fallback($topLevelError, $jobs)
         $result->error = $topLevelError;
         $result->results = array_map(function (\WF\Hypernova\Job $job) {
             $jobResult = new JobResult();
-            $uuid = \Ramsey\Uuid\Uuid::uuid4();
+            $uuid = random_bytes(16);
```

Comment:
This can cause conflicts, should be a uuid

Reply:
The id only need to be unique on every render, not need to be globally unique...